src/commands/generate-odata-client.ts

```diff
@@ -48,18 +48,18 @@ export default class GenerateODataClient extends Command {
       .map(([key, value]) => `--${key}=${value}`);
 
     try {
-      await execa('npm', ['ls', '-g', '@sap/cloud-sdk-generator']);
+      await execa('npm', ['ls', '-g', '@sap-cloud-sdk/generator']);
     } catch ({ exitCode }) {
       if (exitCode === 1) {
         this.log('');
-        this.log('To generate an OData client, it is necessary to install the @sap/cloud-sdk-generator.');
+        this.log('To generate an OData client, it is necessary to install the @sap-cloud-sdk/generator.');
         this.log('For now, the CLI expects the generator to be installed globally.');
         this.log('');
 
-        if (await cli.confirm('Do you want to install the @sap/cloud-sdk-generator globally? (y|n)')) {
-          await execa('npm', ['install', '--global', '--@sap:registry=https://npm.sap.com', '@sap/cloud-sdk-generator']);
+        if (await cli.confirm('Do you want to install the @sap-cloud-sdk/generator globally? (y|n)')) {
+          await execa('npm', ['install', '--global', '--@sap:registry=https://npm.sap.com', '@sap-cloud-sdk/generator']);
```

Comment:
`--@sap:registry=https://npm.sap.com` should be removed otherwise the dependency will not be found

Reply:
I tried to remove this, then the pipeline failed:
https://github.com/SAP/cloud-sdk-cli/runs/497189204?check_suite_focus=true

Then I fixed it by reverting the change:
https://github.com/SAP/cloud-sdk-cli/pull/58/commits/3a136e6785b931c050942d10df600851c8b39846